### PR TITLE
dnsutils package => bind9-dnsutils

### PR DESCRIPTION
**Pull request by mstormi**
as 'dnsutils' no longer is available in Ubuntu.
bind9-dnsutils is there for both OS.

Signed-off-by: Markus Storm <markus.storm@gmx.net>

### CODE REVIEW

**Comment by holgerfriedrich** [comment]
@mstormi bind9-dnsustils is not available in Debian buster (current stable).
It seems we need some OS-dependent package install.

**Comment by mstormi** [comment]
[13:04:52] root@devpi:/var/cache/apt/archives# apt-cache search bind9-dnsutils
bind9-dnsutils - Clients provided with BIND 9

?

**Comment by holgerfriedrich** [comment]
Not there in Raspbian buster. Not there in Debian buster (there you can get it from testing / backports repo).
See apt policy.
So I would suggest to use add a helper function which returns the matching name of the bnsutils package based on OS detection (`is_...` functions).

**Comment by mstormi** [comment]
> Not there in Raspbian buster. Not there in Debian buster (there you can get it from testing / backports repo).
> See apt policy.

Yes in unstable it *is* there. What do you mean by "apt policy" ?

Either way I made it a helper to install either bind9-dnsutils if available and dnsutils else, independent of the OS.

**Comment by mstormi** [comment]
@holgerfriedrich please check again

File: functions/helpers.bash [hunk 1]
```diff
@@ -442,3 +442,17 @@ select_blkdev() {
...
   fi
 }
+## install bind9-dnsutils package if available (currently only in sid and focal)
+## else resort to dnsutils
+##
+##    install_dnsutils()
+##
+install_dnsutils() {
+  if apt-cache show bind9-dnsutils &>/dev/null; then
+    apt-get install bind9-dnsutils
```
Comment (holgerfriedrich) [inline functions/helpers.bash:452]: maybe `--yes` is helpful here

File: functions/helpers.bash [hunk 2]
```diff
@@ -442,3 +442,17 @@ select_blkdev() {
...
+  else
+    apt-get install dnsutils
```
Comment (holgerfriedrich) [inline functions/helpers.bash:454]: maybe `--yes` is helpful here

**Review by holgerfriedrich — COMMENTED** [review]
LGTM besides the comments

**Pull request merged**